src/commands/projects.rs

```diff
@@ -19,7 +19,12 @@ pub fn execute<'a>(matches: &ArgMatches<'a>) -> Result<(), Error> {
     let api = Api::current();
     let org = config.get_org(matches)?;
     let mut projects = api.list_organization_projects(&org)?;
-    projects.sort_by_key(|p| (p.team.name.clone(), p.name.clone()));
+    projects.sort_by_key(|p| {
+        (
+            p.team.as_ref().map_or(String::new(), |t| t.name.clone()),
```

Comment:
It also works with `p.team.as_ref().map_or(None, |t| Some(t.name.clone()))`, but I couldn't find a way to force it being last in the sorted list (other than using something ridiculous like very high unicode character